### DISCUSSION TITLE
MAME for macOS occasionally hang during startup

### DISCUSSION THREAD

**Discussion by wyatt-wong** [mamedev/discussions#149] [General]
I built MAME for macOS using the GitHub source. But I found that when I started MAME from command line, it will occasionally hang with a black screen and my mouse cursor cannot move, why ?

I am using macOS Sequoia 15.0.1

**Comment by danifunker** [comment]
I'm experiencing the same thing on Tahoe. I added some debugging options to my build which is why I built it from source. I think this is still an issue

When it loads the rom I was expecting it to tell me there are compatibility issues with it, but when it hangs that doesn't even appear. 

I believe I compiled it using the brew version of libsdl

**Reply by wyatt-wong** [reply]
I didn't receive that black screen so often now. I just built MAME 0.284 from GitHub source last night using Intel macOS Sequoia 15.7.3.

**Comment by rb6502** [comment]
Try -video opengl if your mame.ini isn't set that way already.
